br/pkg/mock/s3iface.go

```diff
@@ -15,30 +15,30 @@ import (
 	gomock "github.com/golang/mock/gomock"
 )
 
-// MockS3API is a mock of S3API interface
+// MockS3API is a mock of S3API interface.
```

Comment:
Better to update these in another dedicated PR instead of this.

Reply:
Em.. I updated this mock because compile fails, maybe caused by dependency update.